apps/estuary/test/integration/event_table_test.exs

```diff
@@ -20,4 +20,24 @@ defmodule Estuary.EventTableTest do
     assert expected_value == actual_value
     assert expected_table_value == actual_table_value
   end
+
+  test "insert_event inserts an event into the #{@event_stream_table_name} table" do
+    aloha_events =
+      Prestige.execute(
+        "SELECT COUNT(*) from #{@event_stream_table_name} WHERE author = 'Steve Aloha' and create_ts = 5 and data = 'some data' and type = 'some type'"
+      )
+      |> Prestige.prefetch()
+
+    assert aloha_events == [[0]]
+    Estuary.EventTable.insert_event("Steve Aloha", 5, "some data", "some type")
+
+    aloha_events =
+      Prestige.execute(
+        "SELECT COUNT(*) from #{@event_stream_table_name} WHERE author = 'Steve Aloha' and create_ts = 5 and data = 'some data' and type = 'some type'"
+      )
+      |> Prestige.prefetch()
+
+    assert aloha_events == [[1]]
+    Prestige.execute("DELETE from #{@event_stream_table_name}") |> Prestige.prefetch()
```

Comment:
Do we need prefetch here?

Reply:
I know this was a separate card but might be worth moving this to `on_exit` call back since the first test in this file should have done the same thing. Looks like it was just missed